main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -646,3 +657,56 @@ DataTableCellUI.prototype._startEdit = function(elmt) {
     MenuSystem.dismissAll();
   });
 };
+
+function checkNonPrintable(content){
+  var stringIncNonPrintable = "";
+  for (var character = 0; character < content.length; character++) {
+    var unprintableChar = "";
+    if ((content.charAt(character)).charCodeAt(0) == 13) {
+      unprintableChar = "<span class='unprintableCharacaters' style='background-color: orange'><b>CR</b></span>";
+    }
+    else if ((content.charAt(character)).charCodeAt(0) == 10) {
+      unprintableChar = "<span class='unprintableCharacaters' style='background-color: orange'><b>LF</b></span>";
+    } 
+    else if ((content.charAt(character)).charCodeAt(0) == 9) {
+      unprintableChar = "<span class='unprintableCharacaters' style='background-color: orange'><b>HT</b></span>";
+    } 
+    else if ((content.charAt(character)).charCodeAt(0) == 16) {
+      unprintableChar = "<span class='unprintableCharacaters' style='background-color: orange'><b>LS</b></span>";
+    } 
+    else if ((content.charAt(character)).charCodeAt(0) == 32) {
+      unprintableChar = "<span class='unprintableCharacaters' style='background-color: orange'><b>NBSP</b></span>";
+    }
+    stringIncNonPrintable += unprintableChar + content.charAt(character);
+  }
+  return stringIncNonPrintable;
+}
+
+function parseToTextAreaFromDiv() {
+  $(".unprintableCharacaters").remove();
+  $(".data-table-cell-editor-editor").val($(".interfaceForTextArea").text());
+}
+
+function nonPrintableCheckBox(){
+  if ($('#toggle-display-characters').prop('checked')) {
+    var rows = $('.data-table tbody > tr');
+    var columns;
+    for (var i = 0; i < rows.length; i++) {
+      columns = $(rows[i]).find('td>div>span');
+      for (var j = 0; j < columns.length; j++) {
+        var originalContent = $(columns[j]).text();
+        if (originalContent != "") {
+          var updatedContent = checkNonPrintable(originalContent);
+          $(columns[j]).html(updatedContent);
+        }
+      }
+    }
+  }
+  else {
+    $(".unprintableCharacaters").remove();
```

Comment:
typo:  unprintableCharacters
```suggestion
    $(".unprintableCharacters").remove();
```